benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -60,10 +60,10 @@ msgstr ""
 "¡Esta bien! Aún puede ser elegible para nuestro programa. Comuníquese con su "
 "proveedor de tránsito local para obtener asistencia."
 
-msgid "enrollment.pages.index.mst_login.start.header"
+msgid "enrollment.pages.index.mst_login.eligibility_confirmed_item.header"
 msgstr "TODO: Your eligibility is confirmed for an Older Adult benefit"
 
-msgid "enrollment.pages.index.mst_login.start.message"
+msgid "enrollment.pages.index.mst_login.eligibility_confirmed_item.message"
```

Comment:
These were updated to `.heading` and `.details` in the sample data

Reply:
Thank you, fixed in https://github.com/cal-itp/benefits/pull/967/commits/f1f20c4547ed5516f415517bb49e8c819b089b9b